src/module.mk

```diff
@@ -217,9 +217,9 @@ $(COLLWRAP).o : $(COLLWRAP).f90
 $(COLLWRAP).mod :  $(COLLWRAP).f90 $(COLLWRAP).o
 	@true
 
-$(COLLWRAP).f90 : $(COLLWRAP).F90
+$(COLLWRAP).f90 : $(COLLWRAP).cpp
 	$(Q)$(MSG)
-	$(Q)$(FC) -E $< | sed "s/_NL_/\n   /g" > $@
+	$(Q)$(CXX) -E $< | perl -pe 's/_NL_/\n/g' > $@
```

Comment:
We should not rely on `perl` being installed on the target machine. Why does `sed` not work here?

Reply:
@Expander, the problem is that using `macOS` it is impossible to insert newline symbol with `sed`.

Reply:
Hmm, I thought there is a way to do this: 
~~~
sed 's/_NL_/
/'
~~~
I.e. one has to insert an actual newline. Can you try this?

Reply:
I think `sh` will complain about it, let me try.

Reply:
If it does complain in the `Makefile`, a workaround could be to generate the `.F90` file using the `configure` script. I know for sure that it works there, because the `autotools` use the same trick.

Reply:
Yesterday I have found the following workaround: I use `.cpp` file to generate `.f90`.

Reply:
Nice! This works of course. :)

Reply:
Your proposal leads to 
```
/bin/sh: 1: Syntax error: Unterminated quoted string
```
if one uses
```make
	$(Q)$(CXX) -E $< | sed -pe 's/_NL_/
	/g' > $@
```

Reply:
Damn. Could you please try:
~~~
sed 's/_NL_/\
/'
~~~

Reply:
I did it already in 31bf907. It didn't work. Are spaces in the beginning of new line crucial? Maybe I should try do it again but without them

Reply:
Yes, for Fortran spaces at the beginning of a line are crucial.

Reply:
Viva the 1970s! :D